### PR TITLE
[styles] Differentiate between grass and grassland/meadow on high zoom 

### LINKED ISSUE #9399: Grassland is the same colour as Grass on high zooms (z19)
need to make it slightly darker.

example location: https://www.openstreetmap.org/#map=18/51.460677/0.025918
(carto renders them the same too...)

### PR DESCRIPTION
**Pull request by IdeallyGrey**
This PR changes the colors of grassland and meadow at `zoom 16` and above to green2. Grassland and meadow are currently set to green1 above `zoom 14`, previously making it impossible to tell them apart from grass above `zoom 16`, when it is also set to green1. Closes #9399.

Before: (Grassland and grass are rendered the same, giving the appearance a large part of the park is mowed grass)
Screenshot_20241122-142857 (1)

After: (Grassland is darker than grass, showing that parts of the park are not mowed)
Screenshot_20241122-155853

### CODE REVIEW

**Comment by map-per** [comment]
Reusing the `forest` color is a bad idea. Now it's not possible to distinguish `natural=grassland` and `natural=forest` any more. So grassland and meadow should rather use a new color.

(Colors are specified in [data/styles/default/light/colors.mapcss](https://github.com/organicmaps/organicmaps/blob/master/data/styles/default/light/colors.mapcss) for the light mode and [data/styles/default/dark/colors.mapcss](https://github.com/organicmaps/organicmaps/blob/master/data/styles/default/dark/colors.mapcss) for the dark mode)

**Comment by IdeallyGrey** [comment]
Thank you for taking a peek. I believe `forest` is still visible -- green2 is used for both `forest` and `grassland`, albeit at different sets of zoom levels. In the second screenshot, the darker green strip at the top is forest. Would you prefer to increase the contrast between the two to make the distinction clearer?

**Comment by map-per** [comment]
Oh, sorry, I overlooked the zoom levels, so it's the same color as park.

But I would still use a new color, because the `green2` color is visually closer to `natural=scrub` than to `landuse=grass`. I think a color that is just a tiny bit darker than `landuse=grass` would look best. Then grass and grassland would look pretty much the same, but could still be distinguished when they are located next to each other.

**Comment by IdeallyGrey** [comment]
Same park with the new color, which is exactly between green1 and green2
Screenshot_20241123-181531

**Comment by map-per** [comment]
The code looks good to me. 
@organicmaps/styles: What do you think about the design/ color choice?

**Comment by matkoniecz** [comment]
grass styling matching forest styling, even at different zoom level seems very suspect to me

I admit that I also see no great value in distinguishing these two area types

**Comment by IdeallyGrey** [comment]
1. With the new "green7", `grassland` again no longer shares any color with `forest`, which should alleviate the styling issue.
2. I can't speak for the person who opened the issue, but personally I feel it could be a useful distinction. For example, near where I live, there is a park, where part of the land is mowed and maintained grass, while the other half is left wild and cut only once per year. The difference between grass a few inches tall and grass that grows up to chest height could be important for some people wanting to do an activity needing one or the other, ex: finding somewhere for kids to play (without ticks), planning an event, or picking flowers.

Of course, I defer to your judgment.

**Comment by pastk** [comment]
Thanks for the PR!

The first version made grass vs grassland distinguishable, but grassland vs park indistinguishable (same green2 color).

The second version is a bit better, but differences between colors are really small and it introduces yet another color to our already impressive range of 8 green colors :)

There was a long discussion whether we practically need so many hardly distinguishable (and changing according to zoom level!) colors in https://github.com/organicmaps/organicmaps/pull/4758#issuecomment-1465235989 and around. But no-one took the courage (yet) to radically review it.

Textures support will reduce our reliance on colors, but won't solve the visual noise (too much details) issue, actually would make it worse.

Bottom line:
I think this PR is OK :)
It just amends what we have here anyway.
Also it introduces a new color for detailed zooms only, which makes sense (show more details when zooming in).

File: data/styles/default/light/colors.mapcss [hunk 1]
```diff
@@ -52,6 +52,7 @@
 
 @green0: #CFD5A8;
 @green1: #C6CF93;
+@green7: #C1CC89;
```
Comment (pastk) [inline data/styles/default/light/colors.mapcss:55]: It would be better to keep enumeration follow color darkness/intensity.
Call it `@green1b` maybe :)

**Comment by pastk** [comment]
> The first version made grass vs grassland distinguishable, but grassland vs park indistinguishable (same green2 color).

Which actually might be alright as meadows and grasslands are probably not used inside of parks often? More of a countryside wilder thing.

**Comment by pastk** [comment]
Thanks! Could you please rebase into just two commits 1) the actual change & 2) regenerate ?

**Review by pastk — APPROVED** [review]

**Comment by pastk** [comment]
Congrats with your first contribution to OM and welcome to the team!
